docs/pages/docs/guides/images-and-files.mdx

```diff
@@ -0,0 +1,266 @@
+import { Markdown } from '../../../components/Markdown';
+import { Well } from '../../../components/primitives/Well';
+import { RelatedContent } from '../../../components/RelatedContent';
+
+# Images and Files
+
+Keystone [field types](../apis/fields) contain both the `image` and `file`. These two field types can be used to reference, and if required, serve Images and/or Files from Keystone. These field types can be backed by either local or S3 storage.
```

Comment:
```suggestion
Keystone [field types](../apis/fields) contain both the `image` and `file`. These two field types can be used to reference, and if required, serve Images and/or Files from Keystone. These field types can be backed by either local or [Amazon S3 storage](https://aws.amazon.com/s3/).
```
Not sure about this suggestion, but some people may not know what s3 is? A new developer starting out?